crates/edr_provider/src/requests/debug.rs

```diff
@@ -120,3 +121,48 @@ impl From<DebugTraceConfig> for edr_evm::DebugTraceConfig {
         }
     }
 }
+
+/// This is the JSON-RPC Debug trace format
+#[derive(Debug, Clone, serde::Deserialize, serde::Serialize)]
```

Comment:
Do we need both `Deserialize` and `Serialize`? Based on the code, it seems that `serde::Serialize` is sufficient.

Reply:
Yes you are right, thanks for pointing it out. I'll remove it.